### ISSUE TITLE
How to highlighting

### ISSUE THREAD

**Issue by AlexLove77** [elasticquent/Elasticquent#101]
I find that the elasticquent model doesn't include a highlighting object so that I need to get it from `ElasticquentPaginator` manually, is there any way to get it easier or add it as an optional parameter?

**Comment by itbdw** [comment]
Well, I think you can use the getHits() method to get the original content from ES.

fox example
```
{
  total: 49,
  max_score: null,
  hits: [
    {
      _index: "game",
      _type: "game",
      _id: "1829352",
      _score: null,
      _source: {
        id: 1829352,
        title: "SpongeBob Obstacle Odyssey 2",
        content: "xxxxx",
      },
      highlight: {
        title: [
          "<em>SpongeBob</em> Obstacle Odyssey 2"
        ]
      },
      sort: [
        "2"
      ]
    }
  }
```

**Comment by AlexLove77** [comment]
@itbdw Thanks, but seems that Paginator does not support getHits().

**Comment by itbdw** [comment]
@AlexLove77 true, add a config and replace the _source data with highlight data seems to be a good idea.

@timgws

**Comment by AlexLove77** [comment]
#104

**Comment by AlexLove77** [comment]
@itbdw Many thanks, 蛤蛤～

**Comment by nathan-zhu** [comment]
@itbdw when can use highlight in Elasticquent?

**Comment by itbdw** [comment]
@nathan-zhu The author still did not merge the pr. I use my fork for now...

**Comment by nathan-zhu** [comment]
@itbdw 给个连接呗，多谢。

**Comment by itbdw** [comment]
我直接先用的这个，然后自己手动改的一些文件 https://github.com/elasticquent/Elasticquent/pull/104

**Comment by nathan-zhu** [comment]
@itbdw 感谢提供这个高亮功能，我还有个问题也请帮看下，下面这个是多字段搜索ES 语句，并让结果返回多字段的高亮效果，我如何在这里应用下面的查询呢？多谢。

`'{ "query": { "bool": { "must": { "bool": { "should": [ { "wildcard": { "name": "*'.$search.'*" } }, { "wildcard": { "desc": "*'.$search.'*" } } ] } } } }, "highlight": { "pre_tags": ["<em>"], "post_tags": ["</em>"], "fields": { "name": {}, "desc": {} } } }'`

**Comment by itbdw** [comment]
@nathan-zhu 我刚刚看到。。。。试了一下没问题啊。。。两个都高亮了

diff 里只是同一个字段高亮匹配到多个结果的时候只取第一个。
